### ISSUE TITLE
Custom aria-labelledby in material select

### ISSUE THREAD

**Issue by kapalex** [ngx-formly/ngx-formly#1746]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
I am trying to set attributes such as 'aria-labelledby'/'aria-label' on select box, but the value does not appear.  I'm using attributes in templateOptions to set it.

**Expected behavior**
aria-labelledby should be replaced by the value I provide, it works well in input components.

**Minimal reproduction of the problem with instructions**

https://stackblitz.com/edit/ngx-formly-ui-material-kfj3oc?file=app/app.component.ts

**What is the motivation / use case for changing the behavior?**

ARIA properties are important and should be easy to set, only option I found currently is with attributes in templateOptions, if there is some alternative that would help as well.

* **Angular version:** 7.0.X

* **Browser:** all

* **Language:** all

**Comment by kapalex** [comment]
I would wager this is related to   #1265

**Comment by kapalex** [comment]
@aitboudad thanks for the quick fix :)

**Comment by aitboudad** [comment]
released in `5.4.2` version.
